www/source/index.html.slim

```diff
@@ -32,188 +25,195 @@ section.sub-hero
-          anywhere. No rewrite required.
-      p
-        | Give legacy apps the habitat they need to thrive in the modern
-          computing ecosystem. They won't know they've left the data center,
-          and there's no need to tell them.
-      .legacy-section--image
-        img[src="/images/graphics/legacy.svg" alt=""
-          onerror="this.src='/images/graphics/legacy.png'"]
+      .production--features.columns.medium-3.medium-offset-1
+        .production--features-item
+          img.production--icon[src="/images/icons/subhero-manage.svg"
+              onerror="this.src='/icons/subhero-manage.png'" alt=""]
+          h4 Supervisor
+          p
+            | Each service starts with a supervisor process that takes commands
+              and communicates them amongst its peers.
+          a href="/docs/concepts-supervisor/" View docs
+        .production--features-item
+          img.production--icon[src="/images/icons/subhero-results.svg"
+              onerror="this.src='/icons/subhero-manage.png'" alt=""]
+          h4 Isolated Build Environment
+          p
+            | Define your entire app in a plan file and generate reproducble,
+              trustworthy, versioned builds in a clean room environment.
+          a href="/docs/concepts-studio/" View docs
+        .production--features-item
+          img.production--icon[src="/images/icons/subhero-platform.svg"
+              onerror="this.src='/icons/subhero-manage.png'" alt=""]
+          h4 Update Strategies
+          p
+            | Determine how your app should respond when new package versions
+              become available.
+          a href="/docs/run-packages-update-strategy/" View docs
+        .production--features-item
+          img.production--icon[src="/images/icons/subhero-manage.svg"
+              onerror="this.src='/icons/subhero-manage.png'" alt=""]
+          h4 Monitoring
+          p
+            | Check the health of your services and get configuration or
+              dependency version information via a customizable HTTP end point.
+          a href="/docs/run-packages-monitoring/" View docs
 
+      .production--features.columns.medium-3.medium-offset-1
+        .production--features-item
+          img.production--icon[src="/images/icons/subhero-platform.svg"
+              onerror="this.src='/icons/subhero-manage.png'" alt=""]
+          h4 Runtime Support
+          p
+            | Configuration changes and files can be passed and circulated to
+              your app in a single command.
+          a href="/docs/run-packages-overview/" View docs
+        .production--features-item
+          img.production--icon[src="/images/icons/subhero-manage.svg"
+              onerror="this.src='/icons/subhero-manage.png'" alt=""]
+          h4 Clustering Strategies
+          p
+            | Start up your services in a variety of built-in topologies such as
+              leader/follower, standalone and initializer.
```

Comment:
I'm not certain of what "initializer" means in this context, but if it's not something you pair with "standalone" there should be an Oxford comma here to separate them.